### ISSUE TITLE
Midpoint Review - Accessibility Feedback - MHV Medications Pending and Partial statuses

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#99235]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- No major concerns! I think this is all on the right track.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** For partial fills, make sure that multiple instances of the partial fill accordion have some unique identifier, so screen reader users can distinguish between them without needing to explore the surrounding context. Refills currently include a numerical index (Refill 1, Refill 2, etc) and doing the same for partial fills would be perfect. (And it's okay to go Partial Fill, Partial Fill 2, etc if you don't want to have a "1" on the first partial fill).
- [ ] **Should:** For pending medications, I think the icon might make more sense in closer proximity to the "This is a new prescription..." message. That's consistent with other icon usage in these cards and --- most relevant to my interests --- should eliminate the need to include any kind of alternative text for the icon since it'll be immediately followed by the text explanation of why this medication is different.
   
   I'm labeling this one as a "should" since you could leave the icon in place and include alt text with it... but I think moving the icon is the cleaner solution overall. Think of it as a "strong should" 🙂.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
